lyricx.py

```diff
@@ -0,0 +1,80 @@
+"""Python Lyricx Command Line Program to Generate Lyrics of Songs"""
+import sqlite3
+
+from requests import get
+
+import constant
+import lyricx_database
+
+FETCH_TRACK_URL = (f'https://api.musixmatch.com/ws/1.1/track.search?apikey={constant.API_KEY}&s_track_rating=DESC'
+                   '&s_has_lyrics&page_size=1&page=1&q_track=')
+FETCH_TRACK_LYRICS = (f'https://api.musixmatch.com/ws/1.1/track.lyrics.get?apikey={constant.API_KEY}'
+                      f'&track_id=')
```

Comment:
If this is constant variable then move this into constant.py file.
also add error message or success message to this file.